### PR TITLE
moved articles to a carousel

### PR DESCRIPTION
**Pull request by johnreiley**

### CODE REVIEW

File: js/custom.js [hunk 1]
```diff
@@ -11,21 +11,21 @@ $("#menu-toggle").click(function(e) {
 });
 
 // Scrolls to the selected menu item on the page
-$(function() {
-  $('a[href*=#]:not([href=#])').click(function() {
-    if (location.pathname.replace(/^\//, '') == this.pathname.replace(/^\//, '') || location.hostname == this.hostname) {
+// $(function() {
```
Comment (johnreiley) [inline js/custom.js]: this method was preventing the carousel next and prev buttons from working correctly so I commented it out

Comment (nouex) [reply js/custom.js]: go ahead and delete the commented code, it came with the jekyll theme and won't be used.

**Review by nouex — CHANGES_REQUESTED** [review]
Looks good! I like the carousel for mobile, I think we could do something else with it on larger viewports but I'll think about that and see what we do later.  

I left a comment, other than that go ahead and merge.

File: js/custom.js (hunk 1 above)
Comment (nouex) [reply js/custom.js]: Actually, we do use that function once, in the "Let's go" button in the first section but go ahead and get rid of that too.

Comment (johnreiley) [reply js/custom.js]: sounds good

**Pull request merged**